extension/src/background/helpers/dataStorage.ts

```diff
@@ -246,6 +246,16 @@ const migrateSorobanRpcUrlNetwork = async () => {
   }
 };
 
+export const resetAccountSubscriptions = async () => {
```

Comment:
We need to reset this object because of the error handling bug when subscribing to accounts. This has no effect on users since Mercury has not been turned on yet.